configure_cpu-set.sh

```diff
@@ -0,0 +1,13 @@
+#/bin/bash
+
+mkdir /dev/cpuset
+mount -t cgroup -o cpuset cpuset /dev/cpuset
+cd /dev/cpuset
+mkdir cpu0
+echo 0 > cpu0/cpuset.cpus
+echo 0 > cpu0/cpuset.mems
+echo 1 > cpuset.cpu_exclusive
+echo 0 > cpuset.sched_load_balance
+echo 1 > cpu0/cpuset.cpu_exclusive
+echo 1 > cpu0/cpuset.mem_exclusive
+echo $$ > cpu0/tasks
```

Comment:
no mix of underscores and dash - terrible.

Please move this file into analyze/ directory.

Please source the file in each analyzer shell script.

Reply:
What do you mean? I am afraid there is no dash. Alter.
The names are given by the kernel, not me

Reply:
I did this intentionally to show which words belong to each other. Will change it if you want